roles/reproducer/tasks/configure_computes.yml

```diff
@@ -61,3 +64,24 @@
       loop: "{{ cifmw_reproducer_compute_repos }}"
       loop_control:
         label: "{{ item.name }}"
+
+    - name: Ensure rhos-release is configured
+      become: true
+      when:
+        - _async_rhos_release.ansible_job_id is defined
+      block:
+        - name: Ensure async flag still exists
+          register: _async_flag
+          ansible.builtin.stat:
+            path: >-
+              /root/.ansible_async/{{ _async_rhos_release.ansible_job_id }}
```

Comment:
It's a left-over from when I was trying to get the async running, and hit the "you need become if you use it for the async task".
Not sure if we'd ever hit a "missing file". I saw there are some stuff to manually clean the flag, not sure if it may eventually be dropped before we're reaching this step?